doc/developers/performance.rst

```diff
@@ -446,12 +445,24 @@ with ``-pg`` which is rather complicated to get working.
 Fortunately there exist two alternative profilers that don't require you to
 recompile everything.
 
-
 Using valgrind / callgrind / kcachegrind
 ----------------------------------------
 
-TODO
+kcachegrind
+~~~~~~~~~~~
+
+You can use ``yep`` to create a profiling report. ``kcachegrind`` provides
+a graphical environment to visualize this report. However, you need to
+convert ``gperftools`` report into ``kcachegrind`` file::
+
+  # Run yep to profile some python script
+  python -m yep -v benchmarks/my_file.py arcene
 
+  # convert profile.log to callgrind compatible format
+  pprof --callgrind /usr/bin/python ./my_file.py.prof > my_file.py.callgrin
```

Comment:
maybe use a ``which python`` here as many people use anaconda and not the system python?